yellowbrick/features/rankd.py

```diff
@@ -271,20 +452,8 @@ def draw(self, X, **kwargs):
         cb = self.ax.figure.colorbar(mesh, None, self.ax)
         cb.outline.set_linewidth(0)
 
-    def finalize(self, **kwargs):
-        """
-        Finalize executes any subclass-specific axes finalization steps.
-        The user calls poof and poof calls finalize.
-
-        Parameters
-        ----------
-        kwargs: dict
-            generic keyword arguments
+        # Reverse the rows to get the lower left triangle
```

Comment:
Maybe I just can't see it in the review (loc are omitted in the diff), but don't forget to call `super()` that's where the title gets set!